packages/driver/src/cypress/log.ts

```diff
@@ -403,24 +409,6 @@ export class Log {
       this.set('next', null)
     }
 
-    if (!Cypress.isCrossOriginSpecBridge) {
```

Comment:
We no longer "pull" for the snapshot from the secondary origin as it can already "push" snapshots.